config/locales/en.yml

```diff
@@ -415,6 +415,7 @@ en:
       blank_date: Enter %{article} %{attribute}
       dob_future: Enter a %{attribute} that is in the past, for example 31 3 1980
       dob_below_min_age: Enter a date of birth before %{date} – you must be over %{min_age} years old to Apply for teacher training
+      before: Enter a start date that is before the end date
```

Comment:
This is quite a specific validation to the start and end date fields so if it is to be moved in the shared validation errors I think the name should be more descriptive, e.g. `:start_date_before_end_date`

Reply:
Yep, makes sense. Will update.